packages/frontend/flake.nix

```diff
@@ -0,0 +1,81 @@
+{
+  inputs = {
+    nixpkgs.url = "github:nixos/nixpkgs";
+    flake-parts.url = "github:hercules-ci/flake-parts";
+    flake-root.url = "github:srid/flake-root";
+    backend.url = "github:ponkila/HomestakerOS/feat/pkgs-webui?dir=packages/backend";
+  };
+
+  outputs = inputs @ {
+    self,
+    nixpkgs,
+    flake-parts,
+    backend,
+    ...
+  }:
+    flake-parts.lib.mkFlake {inherit inputs;} {
+      systems = [
+        #"aarch64-darwin"
+        #"aarch64-linux"
+        #"x86_64-darwin"
+        "x86_64-linux"
+      ];
+      imports = [
+        inputs.flake-root.flakeModule
+      ];
+
+      perSystem = {
+        pkgs,
+        lib,
+        config,
+        self',
+        inputs',
+        system,
+        ...
+      }: {
+        formatter = nixpkgs.legacyPackages.${system}.alejandra;
+
+        packages.webui = let
+          src = ./.;
```

Comment:
webui should be in project root /webui folder (where it was before) and the source should just import it from there